.github/workflows/tests.yaml

```diff
@@ -0,0 +1,42 @@
+name: Run tests with Tox
+
+on: [push]
+
+jobs:
+  unit-tests:
+    name: Lint, Unit Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.5, 3.6, 3.7, 3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install Tox
+        run: pip install tox
+      - name: Run Tox
+        run: tox # Run tox using the version of Python in `PATH`
+
+  integration-tests:
```

Comment:
I'm not sure that it's worth keeping the existing Build Charm job since this is going to both build and test the charm.

Reply:
That's right, I'll remove build.yaml.

Reply:
Actually, don't worry about this either.  I'll do that in the other PR cleaning up the `script` dir and `Makefile`. Thanks.